Server/Server.Core/Server.Core.Static/Settings.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <map>
+
+namespace Server::Builder
+{
+struct ParamType
+{
+    constexpr static char const* ServerPort  = "--serverport";
+    constexpr static char const* HostAddress = "--hostaddr";
+    constexpr static char const* DBName      = "--dbname";
+    constexpr static char const* DBPort      = "--port";
+    constexpr static char const* DBUser      = "--user";
+    constexpr static char const* DBPassword  = "--password";
+};
+
+class Settings
+{
+public:
+    Settings() = default;
+
+    inline Settings& SetValue(std::pair<std::string, std::string> value)
+    {
+        _settings.try_emplace(value.first, value.second);
+        return *this;
```

Comment:
try_emplace does nothing if the item exists in the map. So here the code can rewrite the old entry. 
Write to the log, at least in the case of this scenario